### ISSUE TITLE
Remove German Geeklog support site from documentation and website

### ISSUE THREAD

**Issue by dhaun** [Geeklog-Core/geeklog#709]
The German Geeklog support site, http://geeklog.info is pretty much dead (last forum post: April 2015). The German Geeklog community is simply too small and probably shrinking.

Since I'm in the process of switching hosting services, I was wondering if it's even worth moving the site over to the new host. I'm coming to the conclusion that it isn't worth the hassle.

I'd therefore suggest that all references to geeklog.info and the existence of a German support site be removed from the Geeklog documentation as well as the geeklog.net website.

ETA: Mid-February 2017

I'm going to keep the domain but make it redirect to geeklog.net. There doesn't appear to be anything worth keeping online at the site (I'll keep local copies just in case).

We can easily set up a German forum section on geeklog.net in case there's demand.

**Comment by eSilverStrike** [comment]
That makes sense to me.

**Comment by mystralkk** [comment]
There are 3 references to "http://geeklog.info" in the documents stored in the GitHub repository.  I will remove or replace them with "https://geeklog.net".

**Comment by mystralkk** [comment]
Changed the 3 occurrences with changeset 4682e01.

**Comment by eSilverStrike** [comment]
Removed geeklog.info references from the support pages and blocks. 

Still a number in articles:

https://www.geeklog.net/search.php?query=geeklog.info&keyType=all&datestart=&dateend=&topic=all&type=stories&author=0&results=30&mode=search

How do you want to handle those?

**Comment by eSilverStrike** [comment]
Okay I am going to close this. I removed all the dead links in the above articles from the search. 

Articles on Geeklog.net go back to 2001.  I am sure there are many more dead links to websites and/or pages that don't exist any more (both external and internal). At some point we should figure out how we want to handle broken links. For articles I modified today I just removed the links but left the text. I did not add a notation or anything that the pages had been updated and the dead links removed.
